<launch>
	<arg name='n' default='1'/>
	<arg name='gui' default='n'/>
	<param name='num_sensors' value="$(arg n)"/>
	
	<node pkg="forcesensor" type="serial_port_wrapper.py" name="port0" output="screen"/>
	
	<node pkg="forcesensor" type="serial_port_wrapper.py" name="port1" output="screen" if="$(eval str(arg('n'))=='2')"/>
	
	<node pkg="forcesensor" type="force_sensor_interface.py" name="force_sensor_gui" output="screen" if= "$(eval arg('gui').lower() == 'y')"/>

	<node pkg="forcesensor" type="sensor.py" name="sensor_obj" output="screen" if= "$(eval arg('gui').lower() == 'n')"/>

</launch>
